cmd/percona-dbaas/main.go

```diff
@@ -33,13 +34,19 @@ var rootCmd = &cobra.Command{
 	Use:   "percona-dbaas",
 	Short: "The simplest DBaaS tool in the world",
 	Long: `    Hello, it is the simplest DBaaS tool in the world,
-    please use commands below to manage your DBaaS.`,
+	please use commands below to manage your DBaaS.`,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		err := detectFormat(cmd)
+		if err != nil {
+			log.Error("detect format:", err)
+		}
+	},
 }
 
 func init() {
 	rootCmd.PersistentFlags().Bool("demo", false, "demo mode (no spinners)")
 	rootCmd.PersistentFlags().MarkHidden("demo")
-
+	rootCmd.PersistentFlags().String("output", "", `Answers format. Can be only "json"`)
```

Comment:
Just a UI issue: `Can be only "json"` is a misleading statement. Because it can be a "text" also.